go.mod

```
@@ -1,6 +1,6 @@
 module github.com/mesosphere/konvoy-image-builder
 
-go 1.18
+go 1.22
```

Let's add toolchain version too to ensure latest is used to build

```suggestion
go 1.22

toolchain go1.22.5
```